release-pipeline.yml

```diff
@@ -0,0 +1,1956 @@
+              downloadPath: $(Build.ArtifactStagingDirectory)
+
+          - task: DownloadBuildArtifacts@0
+            displayName: 'Download Windows Build Artifacts'
+            inputs:
+              artifactName: 'azCopy-windows-signed'
+              downloadPath: $(Build.ArtifactStagingDirectory)
+
+          - task: DownloadBuildArtifacts@0
+            displayName: 'Download Mac Build Artifacts'
+            inputs:
+              artifactName: 'azCopy-mac-signed'
+              downloadPath: $(Build.ArtifactStagingDirectory)
+          
+          - task: DownloadBuildArtifacts@0
+            displayName: 'Download Binaries Build Artifacts'
+            inputs:
+              artifactName: 'azCopy-binaries'
+              downloadPath: $(Build.ArtifactStagingDirectory)
+
+          - script: |
+              
+              sudo apt-get install -y zip
+              cp $(Build.ArtifactStagingDirectory)/azCopy-linux-signed/azcopy-*x86_64.deb .
+              sudo dpkg --info azcopy-*x86_64.deb
+              sudo dpkg -i azcopy-*x86_64.deb
+              sudo apt-get install build-essential -y
+            displayName: 'Create drop.zip file and install azcopy deb package'
+
+          - script: |
+              cp $(work_dir)/NOTICE.txt $(Build.ArtifactStagingDirectory)/azCopy-binaries
+              cd $(Build.ArtifactStagingDirectory)/azCopy-binaries
+              echo "Contents of $(Build.ArtifactStagingDirectory)/azCopy-binaries:"
+              ls -l
+              
+              # Create the zip archive with all files inside azCopy-binaries
+              zip -r $(Build.ArtifactStagingDirectory)/drop.zip .
+              cd $(Build.ArtifactStagingDirectory)
+
+              # List the contents of the resulting zip file for verification
+              unzip -l $(Build.ArtifactStagingDirectory)/drop.zip
+
+              mv $(Build.ArtifactStagingDirectory)/azCopy-linux-signed/mariner .
+            displayName: 'Archive all binaries to drop.zip'
+            workingDirectory: '$(Build.ArtifactStagingDirectory)'
+
+          - task: Bash@3
+            displayName: 'Install Azure CLI'
+            inputs:
+              targetType: 'inline'
+              script: |
+                curl -sL https://aka.ms/InstallAzureCLIDeb | sudo bash
+
+          - task: AzureCLI@2
+            displayName: 'Azure CLI'
+            inputs:
+              azureSubscription: 'ESRP KeyVault identity'
+              scriptType: bash
+              scriptLocation: inlineScript
+              inlineScript: |
+                # Find the current version
+                raw_version_output=$(azcopy --version)
+                azcopy_version=$(echo $raw_version_output | awk '{print $3}')
+
+                today=$(date +"%Y%m%d")
+                
+                if [ $(Test_Mode) = "True" ]; then
+                  container_url="https://$(Test_Release_Account_Name).blob.core.windows.net/releases/release-$azcopy_version-$today"
```

Comment:
is this Test_Release Account Name documented anywhere?

Reply:
initialized Test_Release_Account_Name in variables section. This is the container name 'azcopyprivatedrops'. If we set the Test_Mode flag while triggering the pipeline the release will be published to this container. The existing azcopy release pipeline also uses the same container to test release.